src/Features/Core/Portable/ExternalAccess/UnitTesting/API/NewUnitTestingIncrementalAnalyzer.cs

```diff
@@ -84,5 +87,7 @@ public void LogAnalyzerCountSummary()
         public void Shutdown()
```

Comment:
The nested #if false above can probably be removed.

Reply:
will do.